tests/test_multipart_operations.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from common_ci_utils.random_utils import (
+    generate_random_hex,
+    generate_unique_resource_name,
+    generate_random_files,
+)
+from utility.utils import (
+    check_data_integrity,
+    split_file_data_for_multipart_upload,
+)
+
+log = logging.getLogger(__name__)
+
+
+def test_s3_multipart_operations(
+    account_manager,
+    s3_client_factory,
+    tmp_directories_factory,
+):
+    """
+    Test basic s3 operations using a noobaa bucket:
+    1. Create an account
+    2. Create a bucket using S3
+    3. Write multipart objects to the bucket
+    4. List multipart objects from the bucket
+    5. Download the objects from the bucket and verify data integrity
+    6. Delete the bucket along with objects using S3
```

Comment:
Delete `6.` since we're no longer doing this as part of the test, and also leave a blank line between  the docstring and the enclosing triple quotes

Reply:
Done